paypal/paypal-messages-android
# feat: add standalone modal (work in progress, do not merge)

## Description

Adds standalone modal.

## Screenshots

## Testing instructions

In the demo app, there is now the line "Click here for a modal". Click that and the modal should open.
